### PR TITLE
Add statsd measurements for backend service calls

### PR DESCRIPTION
**Pull request by aub**
This adds a statsd measurement gem and sends measurements to it about our calls to backend services.

### CODE REVIEW

File: lib/breakers/statsd_plugin.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+module Breakers
+  class StatsdPlugin
+    def on_outage_begin(outage)
+      StatsD.gauge("api.external_service.#{outage.service.name}.*.up", 0)
```
Comment (jkassemi) [inline lib/breakers/statsd_plugin.rb]: I removed the "resource" after your feedback in https://github.com/department-of-veterans-affairs/devops/pull/909, so the ".*" can be removed.

File: config/initializers/statsd.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+StatsD.backend = if Rails.env.development? || Rails.env.test?
+                   StatsD::Instrument::Backends::LoggerBackend.new(Rails.logger)
```
Comment (jkassemi) [inline config/initializers/statsd.rb]: This might be worth just tying to a non-blank `STATSD_PORT` and `STATSD_HOST`

**Comment by aub** [comment]
@jkassemi Can you give this a final pass when you have a sec?

**Comment by jkassemi** [comment]
Looks good - when tests pass (queued behind several `vets-website` builds) this is good for merge at your convenience.

**Pull request merged**